app/plugins/Wallet/js/buttons.js

```diff
@@ -11,6 +11,22 @@ eID('create-address').onclick = function() {
 	IPC.sendToHost('api-call', call, 'new-address');
 };
 
+// Filt address list by search string
+eID('search-address').onclick = function() {
+	tooltip('Searching...', this);
+	var searchstr = eID('search-bar').value;
+
+	NodeList.prototype.forEach = Array.prototype.forEach
+	var entries = eID('address-list').childNodes;
+	entries.forEach( function(entry) {
+		if (entry.getElementsByClassName('address')[0].innerHTML.indexOf(searchstr)) {
```

Comment:
`querySelector` will get you the first element you need, negating the need to have `[0]`